src/obsidian_to_org/__main__.py

```diff
@@ -12,11 +12,11 @@
 
 COMMENT_MARKER = "#!#comment:"
 RULER_RE = re.compile(r"^---\n(.+)", re.MULTILINE)
-LINK_RE = re.compile(r"\[\[([^|\[]*?)\]\]")
+LINK_RE = re.compile(r"\[\[([^|\\/:#*^\[\]]*?)\]\]")
```

Comment:
👋  What do the revised regex rules accomplish? I'm not too familiar with the syntax and there isn't a description in the PR.

Reply:
This is a full list of forbidden characters in Obsidian file name.